pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.java

```diff
@@ -68,6 +80,28 @@ default GovernancePolicy createPolicy(GovernancePolicy policy) {
     @UseRowReducer(PolicyWithAgeOfMajorityRuleReducer.class)
     Optional<GovernancePolicy> findPolicyByStudyGuid(@Bind("studyGuid") String studyGuid);
 
+    default void addAgeUpCandidate(long studyId, long participantUserId) {
+        getStudyGovernanceSql().insertAgeUpCandidate(studyId, participantUserId);
+    }
+
+    default void markAgeUpPrepInitiated(Set<Long> candidateIds) {
```

Comment:
For `markAgeUpPrepInitiated()` and `removeAgeUpCandidates()`, can you pass back the actual rowcounts and use these in the log statements around `AgeUpService:L97`?  They _should_ always match, but if we're logging what was done, let's bank on whatever the database says.